gmm_client/src/client_requests.erl

```diff
@@ -0,0 +1,36 @@
+%%%-------------------------------------------------------------------
+%%% @author Piotr Świderski
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 10. maj 2021 16:54
+%%%-------------------------------------------------------------------
+-module(client_requests).
+-author("Piotr Świderski").
+
+
+%% API
+-export([post_request/1, get_delete_request/2,
+         get_simple_request_body/1]).
+
+request(Method, Url, Headers, Payload, Options )->
+  {ok,_,_,_} = hackney:request(Method, Url, Headers, Payload, Options).
+
+simple_request(Method, Url) -> request(Method, Url, [], <<>>, []).
+
+get_simple_request_body(Url)->
+  {ok,_, _, ClientRef} = simple_request(get, Url),
+  hackney:body(ClientRef).
+
+post_request(Path)->
+  ReqHeaders = [{<<"Content-Type">>, <<"application/json">>}],
+  hackney:request(post, Path, ReqHeaders, <<>>, []).
+  % potencjalnie do przyszego uzytku // w argumentach wtedy Body
+  %ok  = hackney:send_body(ClientRef, Body),
+  %{ok, _Status, _Headers, ClientRef} = hackney:start_response(ClientRef),
+  %hackney:body(ClientRef).
+
+get_delete_request(Url, Method)->
```

Comment:
wiesz co, skoro w kliencie robisz oddzielne funkcje do tworzenia grup, userów i providerów, to moze tą funkcję też podziel na dwie, żeby w kodzie to ładniej wyglądało

Reply:
i wciągnij do funkcji do getów list_body_elements od razu przy okazji, skoro i tak w kliencie używamy tego w każdym gecie

Reply:
znaczy tutaj jest o tyle fajne, że przekazujesz metodę, być może powinna być inna nazwa, ale sama forma jest moim zdaniem bardzo wygodna

Reply:
list body elements jest używana seryjne dla obecnego modelu